### PR TITLE
- fixed get_orchestrator_version()

### PR DESCRIPTION
**Pull request by dmitsh**
- added logs to dcos and swarmmode validation

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/639)

### CODE REVIEW

File: test/common.sh [hunk 1]
```diff
@@ -167,6 +167,9 @@ function get_orchestrator_version() {
 	[[ ! -z "${CLUSTER_DEFINITION:-}" ]] || (echo "Must specify CLUSTER_DEFINITION" && exit -1)
 
 	orchestratorVersion=$(jq -r 'getpath(["properties","orchestratorProfile","orchestratorVersion"])' ${CLUSTER_DEFINITION})
+	if [[ "$orchestratorVersion" == "null" ]]; then
```
Comment (JiangtianLi) [inline test/common.sh]: where is "null" from? does it need to be case-insensitive?

File: test/cluster-tests/swarmmode/test.sh [hunk 2]
```diff
@@ -6,6 +6,18 @@ set -u
 
 ssh_args="-i ${SSH_KEY} -o StrictHostKeyChecking=no -p2200 azureuser@${INSTANCE_NAME}.${LOCATION}.cloudapp.azure.com"
 
+function log {
```
Comment (JiangtianLi) [inline test/cluster-tests/swarmmode/test.sh]: Can this function be moved to a common sh such as util.sh or common.sh?

**Comment by JiangtianLi** [comment]
LGTM

**Comment by dmitsh** [comment]
@acs-bot test this please

**Comment by dmitsh** [comment]
@acs-bot test this please

**Pull request merged**